packages/lesswrong/components/posts/TableOfContents/DynamicTableOfContents.tsx

```diff
@@ -10,8 +10,9 @@ export interface DynamicTableOfContentsContextType {
 
 export const DynamicTableOfContentsContext = React.createContext<DynamicTableOfContentsContextType | null>(null);
 
-export const DynamicTableOfContents = ({title, children}: {
+export const DynamicTableOfContents = ({title, rightColumnChildren, children}: {
   title?: string,
+  rightColumnChildren?: React.ReactNode[],
```

Comment:
I think instead of being an array, rightHandChildren should just be an object (similar to "children" in most wrapper components), and if you want to submit multiple children, just wrap them both in a div.